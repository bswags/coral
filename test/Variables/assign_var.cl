#cordbconn
server="server";
user="server";
password="server";
port="server";
DBName="server";
type="sqlite";
#enddbconn

#cordb
#enddb
void main() {

    int a = 1;
    int b = 2;
    printf(a+b);

}