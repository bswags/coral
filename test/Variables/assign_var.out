Traceback (most recent call last):
  File "Variables/assign_var.clx", line 3, in <module>
    import coral_backend
ImportError: No module named coral_backend
